Doc/library/inspect.rst

```diff
@@ -581,6 +581,9 @@ function.
    Raises :exc:`ValueError` if no signature can be provided, and
    :exc:`TypeError` if that type of object is not supported.
 
+   ``globalns`` and ``localns`` are the namespaces that are passed into
+   :func:`typing.get_type_hints` when resolving the annotations.
```

Comment:
```suggestion
   ``globalns`` and ``localns`` are passed into
   :func:`typing.get_type_hints` when resolving the annotations.
```
(Can't say they are namespaces because they may be None. But since we just pass them through it's probably okay to say just that.)